govcd/region.go

```diff
@@ -0,0 +1,100 @@
+package govcd
```

Comment:
Just noticed, should we name TM related files with `tm_` prefix?

Reply:
renamed region and supervisor files to have `tm_` prefix